src/containers/AlbumsView.jsx

```diff
@@ -32,17 +32,18 @@ export class AlbumsView extends Component {
       return <ErrorComponent errorType='albums' />
     }
     return (
-      <div>
-        <Topbar viewName='albums' />
-        <AlbumsList {...this.props} {...this.state} />
-        { this.props.children }
-      </div>
+      !this.props.children
```

Comment:
Due to the `AppRoute` component, the `albumPhotos` container will be a child of this container, it's how the routes work for now: 
```javascript
<Route path='albums' component={AlbumsView}>
    <Route path=':albumId' component={AlbumPhotos}>
         <Route path=':photoId' component={Viewer} />
    </Route>
</Route>
```

I could by pass this way by declaring differently:
```javascript
<Route path='albums' component={AlbumsView}/>
<Route path='albums/:albumId' component={AlbumPhotos}/>
<Route path='albums/:albumId/:photoId' component={Viewer} />
```
But I will lose the semantic of these routes :/ and the store will behave differently as well.